app/models/payment/go_cardless/transaction.rb

```diff
@@ -81,8 +81,10 @@ def publish_failed_subscription_charge
     ChampaignQueue.push({
       type: 'subscription-payment',
       params: {
+        created_at: created_at,
         recurring_id: subscription.go_cardless_id,
-        success: 0
+        success: 0,
+        status: 'failed'
```

Comment:
I see. Anyway, if `status` is something that's only relevant to AK then I think it should be added in the worker. Managing AK specific syntax and details is the worker's responsibility.